mungegithub/github/github.go

```diff
@@ -726,12 +726,17 @@ func (obj *MungeObject) AddLabel(label string) error {
 func (obj *MungeObject) AddLabels(labels []string) error {
 	config := obj.config
 	prNum := *obj.Issue.Number
-	config.analytics.AddLabels.Call(config, nil)
 	glog.Infof("Adding labels %v to PR %d", labels, prNum)
```

Comment:
This needs to be after the `IsSuperset()` early return (like you already did with the analytics collection)